src/components/Dialog/Dialog.tsx

```diff
@@ -288,7 +288,7 @@ export default class Dialog extends Component<DialogProps, DialogState> {
    */
   buildAssetWithFullUrl(asset: AssetProps[]) {
     const scheme = 'https://';
-    const domain = this.state.selectedSource.name;
+    const domain = this.state.selectedSource.domain;
```

Comment:
Triply checking that this does in fast always return a string. Vaguely remember we parse the source domains by selecting the first index, so I think we're good. I want to be sure.

Reply:
Yes I think this is guaranteed because it's impossible (to my knowledge) to create a Source without having at least one subdomain. I guess we can take an extra precaution here and fallback to the `selectedSource.name` if a `domain` does not exist.